api/rhsm-subscriptions-api-spec.yaml

```diff
@@ -1140,7 +1140,6 @@ components:
       type: string
       enum:
         - OpenShift Container Platform
-        - RHEL
```

Comment:
How about changing examples of responses at other places from api/rhsm-subscriptions-api-spec.yaml ?
 
```
first: '/api/rhsm-subscriptions/v1/hosts/RHEL?sla=Premium&usage=Production&offset=0&limit=5'
last: '/api/rhsm-subscriptions/v1/hosts/RHEL?sla=Premium&usage=Production&offset=5&limit=5'
previous: null
next: '/api/rhsm-subscriptions/v1/hosts/RHEL?sla=Premium&usage=Production&offset=5&limit=5'
```

Reply:
Yeah good catch will update!